index.html

```diff
@@ -1,203 +1,294 @@
-                      <li>
-                        <p>CSS</p>
-                      </li>
-                    </ul>
-                  </div>
-                  </div>
-                    <br>
-                      <div class="content-sec">
-                      <label>Frameworks</label>
-                      <img src="image/icon-right.png" alt="icon" class="icon-right">
+            <div class="programming-sec">
+              <ul class="outer-item">
+                <li class="programming">
+                  <img src="image/html.png" alt="HTML" />
+                </li>
+                <li>
+                  <p>HTML</p>
+                </li>
+              </ul>
+            </div>
+            <div class="programming-sec">
+              <ul class="outer-item">
+                <li class="programming">
+                  <img src="image/css.png" alt="CSS" />
+                </li>
+                <li>
+                  <p>CSS</p>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <br />
+          <div class="content-sec">
+            <label>Frameworks</label>
+            <img src="image/icon-right.png" alt="icon" class="icon-right" />
+          </div>
+          <div class="underline"></div>
+          <div class="content-sec">
+            <label>Skills</label>
+            <img src="image/icon-right.png" alt="icon" class="icon-right" />
+          </div>
+          <div class="underline"></div>
+        </div>
+      </section>
+    </main>
 
-                  </div>
-                  <div class="underline"></div>
-                  <div class="content-sec">
-                      <label>Skills</label>
-                          <img src="image/icon-right.png" alt="icon" class="icon-right">
-                        </div>
-                  <div class="underline"></div>
-                </div>
-              </section>
-        </main>
+    <footer class="footer">
+      <div class="contact-content">
+        <h2 class="contact-header">Contant Us</h2>
+        <p class="contact-paragraph">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <form action="https://formspree.io/f/xjvlllyy" method="POST">
+        <input
+          name="name"
+          id="user-name"
+          type="text"
+          placeholder="Name"
+          maxlength="30"
+          required
+        />
 
-</body>
+        <input
+          name="email"
+          id="user-email"
+          type="email"
+          placeholder="name@gmail.com"
+          required
+        />
 
+        <textarea
+          name="message"
+          maxlength="500"
+          required
+        >
+        Write your message here
+        </textarea>
+        <button type="submit">Get in Touch</button>
```

Comment:
- [ ] The submit button should be aligned to the left of the form, please pay more attention to details and try to creat the Figma design as close as possible

|Your design|Figma design|
|-|-|
|![Screenshot_20220531_212058](https://user-images.githubusercontent.com/25227022/171278261-abfe0008-bb5d-4d49-aeef-2dd8b2f6db48.png)|![Screenshot_20220531_212428](https://user-images.githubusercontent.com/25227022/171278284-75ada8a9-4a03-4d7b-b62a-aeee15ae7df9.png)|